internal/compiler/widgets/material-base/widget-checkbox.slint

```diff
@@ -22,45 +24,49 @@ export component CheckBox {
     layout := HorizontalLayout {
         spacing: 16px;
 
-        Rectangle {
-            width: 18px;
-            height: 18px;
-
-            container := Rectangle {
-                width: 100%;
-                height: 100%;
-                border-radius: 2px;
-                border-width: 2px;
-                border-color: md.sys.color.on-surface;
-            }
-
-            state-layer := Rectangle {
-                width: 40px;
-                height: 40px;
-                x: (parent.width - self.width) / 2;
-                y: (parent.height - self.height) / 2;
-                opacity: 0;
-                background: md.sys.color.primary;
-                border-radius: 20px;
-                animate opacity { duration: 300ms; easing: ease; }
-            }
-
-            if (root.checked) : Path {
-                width: 66%;
-                height: 66%;
-                x: (parent.width - self.width) / 2;
-                y: (parent.height - self.height) / 2;
-                commands: "M18.9 35.7 7.7 24.5l2.15-2.15 9.05 9.05 19.2-19.2 2.15 2.15Z";
-                fill: md.sys.color.on-primary;
-
-                states [
-                    disabled when !root.enabled : {
-                        fill: md.sys.color.on-surface;
-                    }
-                    hover-selected when touch.has-hover && root.checked : {
-                        fill: md.sys.color.on-primary;
-                    }
-                ]
+        VerticalLayout {
```

Comment:
In the PR description this is mentioned as "Fix layout" but perhaps this is worth mentioning in the ChangeLog for our users as well? What was the issue?

Reply:
The issue was only specific to `Switch` from the handling of the `GroupBox` by user it should not bring changes.

Reply:
I'm still curious about this one. Does this improve the situation when putting a `CheckBox` into a `GroupBox` or a `VerticalLayout` in general? If yes, isn't that worth mentioning in the ChangeLog?

Reply:
The problem was that I have issues with the space with the padding for the `Switch`. I had this problem only with the Switch and not the CheckBox